app/controllers/users_controller.rb

```diff
@@ -5,6 +5,7 @@ def new
   end
 
   def create
+    redirect_to videos_path if logged_in?
```

Comment:
Yes, there is no need for this, but it is not bad either. You can leave it if you like it.